config/crd/patches/webhook_in_opentelemetrycollectors.yaml

```diff
@@ -2,15 +2,16 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: opampbridges.opentelemetry.io
+  name: opentelemetrycollectors.opentelemetry.io
```

Comment:
why does it think its changing the opampbridge CRD here?

Reply:
I changed this manually. The file was not used anyways.